www/capnew.php

```diff
@@ -297,7 +297,7 @@
 define('LIME', '#C7FF00');  // RGB 199 255 0
 define('GREEN', '#00BE00'); // 0 190 0
 
-define('POBOX','Hangar 10 Airfield Avenue, Murwillumbah NSW 2484, New South Wales, (Commonwealth of) Australia');
+define('POBOX','Clos Belmont 2, 1208 Genève, Suisse');
```

Comment:
Suisse or Switzerland?

Reply:
If it is written in French: Genève, Suisse.
(actually, you could also write in German (Genf, Schweiz) or Italian (Ginevra, Svizzera))